backends/influx.py

```diff
@@ -25,15 +26,17 @@ def _query_assemble(clause, namespace, start, end, field=None,
     elif clause.upper() == 'DELETE':
         clause += f' FROM "{namespace}"'
     else:
-        log.error(f'Error. Invalid clause "{clause}".')
+        error = f'Error. Invalid clause "{clause}".'
+        log.error(error)
+        raise BackendError(error)
 
     time_clause = ' WHERE time '
     if start is not None:
         clause += f'{time_clause} >= \'{str(start)}\''
         if end is not None:
             clause += f' AND time <=\'{str(end)}\''
     elif start is None and end is not None:
-        clause += f'{time_clause} <= \'{str(end)}\''
+        clause += f'{time_clause}<= \'{str(end)}\''
```

Comment:
I'd suggest replacing outside single quotes for double quotes to avoid the use of backslashes. Not only in this line, but also the ones above it.

```suggestion
        clause += f"{time_clause} <= '{str(end)}'"
```

Reply:
Yeah.. It will be more readable.